extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaTuningConfig.java

```diff
@@ -210,54 +228,34 @@ public boolean equals(Object o)
     if (o == null || getClass() != o.getClass()) {
       return false;
     }
-
     KafkaTuningConfig that = (KafkaTuningConfig) o;
-
-    if (maxRowsInMemory != that.maxRowsInMemory) {
-      return false;
-    }
-    if (maxRowsPerSegment != that.maxRowsPerSegment) {
-      return false;
-    }
-    if (maxPendingPersists != that.maxPendingPersists) {
-      return false;
-    }
-    if (reportParseExceptions != that.reportParseExceptions) {
-      return false;
-    }
-    if (handoffConditionTimeout != that.handoffConditionTimeout) {
-      return false;
-    }
-    if (resetOffsetAutomatically != that.resetOffsetAutomatically) {
-      return false;
-    }
-    if (intermediatePersistPeriod != null
-        ? !intermediatePersistPeriod.equals(that.intermediatePersistPeriod)
-        : that.intermediatePersistPeriod != null) {
-      return false;
-    }
-    if (basePersistDirectory != null
-        ? !basePersistDirectory.equals(that.basePersistDirectory)
-        : that.basePersistDirectory != null) {
-      return false;
-    }
-    return indexSpec != null ? indexSpec.equals(that.indexSpec) : that.indexSpec == null;
-
+    return maxRowsInMemory == that.maxRowsInMemory &&
```

Comment:
am wondering if this is autogenrated or handmade?

Reply:
Generated